### PR TITLE
Provider-controlled invitation expiry

### PR DESCRIPTION
**Pull request by smangelsdorf**
Allows the expiry time of an invitation to be set by the provider (via web or API)

Re: #45

### CODE REVIEW

File: app/views/invitations/new.html.erb [hunk 1]
```diff
@@ -20,6 +20,11 @@
     <%= f.text_field(:mail) %>
   <%- end -%>
 
+  <%= field_block do -%>
```
Comment (rianniello) [inline app/views/invitations/new.html.erb]: No client side validation? Got this if I put in a blank value:

image

**Comment by rianniello** [comment]
We're allowed to create an invitation with an expiry earlier than today? I guess this invitation will never be able to be used.

Reckon it's worth adding validation? I'm not overly phased about it, just thought i'd mention.

**Comment by smangelsdorf** [comment]
Done.

**Comment by smangelsdorf** [comment]
Done.

**Comment by smangelsdorf** [comment]
Alright, finally got to the bottom of those weird test failures.

Before you run the tests again, make sure to `brew install phantomjs`

**Comment by rianniello** [comment]
All looks good.

**Pull request merged**